packages/react-codemirror-playground/src/tree-util.ts

```diff
@@ -25,5 +26,11 @@ export function getDebugTree(cypher: string): SimpleTree {
     };
   }
 
-  return walk(statements);
+  const children = statements.map((statement) => walk(statement));
+
+  return {
+    // TODO Is this correct?
+    name: 'root',
```

Comment:
Should we name this differently?

Reply:
We've got a similar pattern in the `parserAdapter` (making up a new node that serves only as a root), there we call it `topNode`. I think it makes sense to call both the same, but I'll leave it to you to decide if that's `root`, `virtualRoot`, `topNode` or some other name

Reply:
Done